apps/api/src/app/environments-v2/usecases/sync-strategies/normalizers/layout.normalizer.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@nestjs/common';
+import { INormalizedLayout } from '../types/layout-sync.types';
+import { LayoutResponseDto } from '../../../../layouts-v2/dtos';
+
+@Injectable()
+export class LayoutNormalizer {
+  /**
+   * We want to normalize the layout and omit any fields that are autogenerated by the system
+   * Or are not relevant for the comparison
+   */
+  normalizeLayout(layout: LayoutResponseDto): INormalizedLayout {
+    const { _id, updatedAt, updatedBy, createdAt, slug, isDefault, origin, type, variables, ...normalizedLayout } =
```

Comment:
I wonder, if we want to recognize here if the default layout is changed, do we allow changing the default? In that case I would expect the field to be there

Reply:
no we don't allow changing the default

Reply:
Got ya 💪